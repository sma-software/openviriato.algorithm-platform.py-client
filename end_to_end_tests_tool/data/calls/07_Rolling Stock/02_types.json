{
    "restResource": "/vehicles/types/{vehicleTypeId}",
    "httpMethod": "GET",
    "urlParameters": [ "4107" ],
    "requestBody": null,
    "CSharpMethodName": "GetVehicleType(vehicleTypeID)",
    "CSharpMethodParameterTypes": [ "System.Int64" ],
    "subcategory" : "Get Vehicle Type" 
}
