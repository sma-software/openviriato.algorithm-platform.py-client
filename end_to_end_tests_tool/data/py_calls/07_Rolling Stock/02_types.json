{
  "PythonParameterMapping": {
    "vehicle_type_id": "$.urlParameters.[0]"
  }
}