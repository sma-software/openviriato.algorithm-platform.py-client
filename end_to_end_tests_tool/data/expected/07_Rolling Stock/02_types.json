{
  "id": 4107,
  "debugString": "vehicle isEngine: True, model: ICN, series: 1, velMax: 200, seats: 125 326",
  "isEngine": true,
  "placesFirstClass": 125,
  "placesSecondClass": 326
}